app/src/components/widgets/charts/investigatedTrendChart/common/utils.js

```diff
@@ -0,0 +1,33 @@
+import * as COLORS from 'common/constants/colors';
+
+const getRange = (start, stop, step) => {
```

Comment:
Please check `components/widgets/charts/common/utils/utils.js` for already existing utils

Reply:
removed